test/hotspot/jtreg/serviceability/dcmd/thread/VThreadSummaryTest.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8337199
+ * @summary Basic test for jcmd Thread.vthread_summary
+ * @requires vm.continuations
+ * @modules jdk.jcmd
+ * @library /test/lib
+ * @run junit/othervm VThreadSummaryTest
+ */
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.net.SocketTimeoutException;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.ForkJoinWorkerThread;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.IntStream;
+import java.lang.management.ManagementFactory;
+import jdk.management.VirtualThreadSchedulerMXBean;
+
+import jdk.test.lib.dcmd.PidJcmdExecutor;
+import jdk.test.lib.process.OutputAnalyzer;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class VThreadSummaryTest {
+
+    private OutputAnalyzer jcmd() {
+        return new PidJcmdExecutor().execute("Thread.vthread_summary");
+    }
+
+    /**
+     * Test that output includes the default scheduler and timeout schedulers.
+     */
+    @Test
+    void testSchedulers() {
+        // ensure default scheduler are timeout schedulers are initialized
+        Thread.startVirtualThread(() -> { });
+
+        jcmd().shouldContain("Virtual thread scheduler:")
+                .shouldContain(Objects.toIdentityString(defaultScheduler()))
+                .shouldContain("Timeout schedulers:")
+                .shouldContain("[0] " + ScheduledThreadPoolExecutor.class.getName());
```

Comment:
Again please align dots on chained calls.

Reply:
and the IDE will of course re-align it at next edit :-)